llvm/test/MC/Mips/macro-la-local.s

```diff
@@ -0,0 +1,23 @@
+# RUN: llvm-mc %s -triple=mips-unknown-linux -show-encoding -mcpu=mips32r2 | \
```

Comment:
remove `-unknown-linux` whenever applicable, to emphasize that this is generic ELF behavior.